packages/styled-system/src/config/border.js

```diff
@@ -1,5 +1,6 @@
 import system from '../core/system';
 import positiveOrNegativeTransform from '../transforms/positiveOrNegative';
+import compoundThemeValueTransform from '../transforms/compoundBorderValue';
```

Comment:
Consider renaming it to `compoundBorderRadiusTransform` or `compoundCornerTransform` if it is only intended to handle `border-radius` values in clockwise order.

For properties like `margin`, `padding`, `border-width`, `border-style`, `inset`, etc., we can implement a separate `compoundEdgeTransform` if we want to support them in the future.